flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -5,12 +5,14 @@ import 'package:flutter/services.dart';
 import 'package:flutter_local_notifications_platform_interface/flutter_local_notifications_platform_interface.dart';
 import 'package:timezone/timezone.dart';
 
+import '../flutter_local_notifications.dart';
```

Comment:
Shouldn't need to import this as that's what contains references to other files. If this was needed then, it tends to be an indication that something was missed